tests/integration/test_pip_packages.py

```diff
@@ -28,14 +28,7 @@ def test_all_pip_packages(env_name, test_env, tmpdir):
     env_data = test_env["pip_packages"][env_name]
     client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
     initial_response = client.create_new_request(
-        payload={
-            "repo": env_data["repo"],
-            "ref": env_data["ref"],
-            "pkg_managers": ["pip"],
-            # TODO: delete pip-dev-preview flag when
-            #  the pip package manager will be ready for production usage
-            "flags": ["pip-dev-preview"],
-        },
+        payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"],}
```

Comment:
```suggestion
        payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"]}
```
`black` is complaining about this

Reply:
Yes I noticed just now. fixed it